src/main/java/teammates/common/util/Const.java

```diff
@@ -767,7 +767,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String STUDENT_SHORT_NAME = "studentshortname";
         public static final String STUDENT_PROFILE_EMAIL = "studentprofileemail";
         public static final String STUDENT_PROFILE_INSTITUTION = "studentprofileinstitute";
-        public static final String STUDENT_NATIONALITY = "studentnationality";
+        public static final String STUDENT_NATIONALITY = "studentNationality";
```

Comment:
I am going to change it back, but then there will be 2 possible options:

1. In the `studentProfileDiv.tag` the `select id` should be written by hand and the value has to be `studentNationality`.
2. In the `StudentProfilePage.java` I have to change the line `@FindBy(id = "studentNationality")`  as well. This will break the naming convention in that file.

Could you please tell me which one do you prefer?